pkg/client/client.go

```diff
@@ -174,3 +174,50 @@ func (c *SPDKClient) EngineGet(name string) (*api.Engine, error) {
 	}
 	return api.ProtoEngineToEngine(resp), nil
 }
+
+func (c *SPDKClient) DiskCreate(diskName, diskPath string, blockSize int64) (*spdkrpc.Disk, error) {
+	if diskName == "" || diskPath == "" {
+		return nil, fmt.Errorf("failed to create disk: missing required parameter")
+	}
+
+	client := c.getSPDKServiceClient()
+	ctx, cancel := context.WithTimeout(context.Background(), GRPCServiceTimeout)
+	defer cancel()
+
+	return client.DiskCreate(ctx, &spdkrpc.DiskCreateRequest{
+		DiskName:  diskName,
+		DiskPath:  diskPath,
+		BlockSize: blockSize,
+	})
+}
+
+func (c *SPDKClient) DiskGet(diskName, diskPath string) (*spdkrpc.Disk, error) {
+	if diskPath == "" {
```

Comment:
Do we need to validate `diskName` as well?

Reply:
Yeah, forgot addint it.
Added, thank you.